src/Html/Events/Extra.elm

```diff
@@ -265,3 +278,31 @@ constructor when an `input`, `select`, or `textarea` element has changed.
 onChange : (String -> msg) -> Attribute msg
 onChange onChangeAction =
     on "change" <| Json.map onChangeAction targetValue
+
+
+{-| Detect input events on multi-choice select elements.
+
+It will grab the string values of `event.target.selectedOptions` on any input
+event.
+
+Check out [`targetSelectedOptions`](#targetSelectedOptions) for more details on
+how this works.
+
+Note: `Html.Events.onInput` parses `event.target.value` that doesn't work with
+multi-choice select elements.
+
+Note 2:
+[`selectedOptions`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/selectedOptions)
+is not supported by Internet Explorer.
+
+-}
+onMultiSelect : (List String -> msg) -> Attribute msg
+onMultiSelect tagger =
+    stopPropagationOn "input" <|
```

Comment:
```suggestion
    stopPropagationOn "change" <|
```

Can we use `change` instead of `input` to bump up the compatibility range for Edge?